librz/core/cconfig.c

```diff
@@ -3230,6 +3230,9 @@ RZ_API int rz_core_config_init(RzCore *core) {
 	SETI("bin.baddr", -1, "Base address of the binary");
 	SETI("bin.laddr", 0, "Base address for loading library ('*.so')");
 	SETCB("bin.dbginfo", "true", &cb_bindbginfo, "Load debug information at startup if available");
+	SETCB("bin.dbginfo.dwo_path", "", NULL, "Load separate debug information (DWARF) file if available");
+	SETCB("bin.dbginfo.debug_file_directory", "/usr/lib/debug", NULL,
```

Comment:
Does it support multiple paths?

Reply:
Yes, multiple paths can be separated by `;`.